service/src/test/java/uk/gov/hmcts/reform/fpl/utils/OrderIssuedNotificationTestHelper.java

```diff
@@ -41,22 +41,17 @@ private OrderIssuedNotificationTestHelper() {
             "orderType", orderType.toLowerCase(),
             "callout", withCallout ? callout : "",
             "courtName", EXAMPLE_COURT,
-            "documentLink", jsonFileObject.toMap(),
+            "documentLink", jsonFileObject,
```

Comment:
hmm, something smells here. In OrderIssuedNotifyData there is comment
```
private final Object documentLink; //This will be String or Map<String, Object> depending on case access/no access
```
and now expected object contains JsonObject and not map